### ISSUE TITLE
Mobile-friendly form creation

### ISSUE THREAD

**Issue by paulshalikashvilli** [formtools/core#380]
Is there any documentation or advice on creating a second, mobile-friendly version of the same form?

**Comment by benkeen** [comment]
No, the documentation - and the Form Builder templates themselves - are a little dated on that. Form Tools 3.2 will be devoted to updating the Form Tools UI for responsiveness so it can be used on a phone & other screen sizes. Updating the Form Builder templates to run on phones will come as a result of that.
